MeshLib/Elements/Element.cpp

```diff
@@ -186,4 +186,20 @@ bool Element::isBoundaryElement() const
         [](MeshLib::Element const*const e){ return e == nullptr; });
 }
 
+#ifndef NDEBUG
+std::ostream& operator<<(std::ostream& os, Element const& e)
+{
+	os << "Element " << e._id << " @ " << &e
```

Comment:
I'm adding a MeshElemType (line, hex) , not the CellType (line2, hex8,...). The number of nodes is visible anyway...